examples/crypto.toit

```diff
@@ -0,0 +1,204 @@
+
+import crypto.aes
+import expect
+
+
+/**
+Randomly generated key with 256 bits.
+Generated from https://www.allkeysgenerator.com/Random/Security-Encryption-Key-Generator.aspx.
+*/
+secret32 ::= #[
```

Comment:
We typically use CONSTANT_NAME for constant names.

Reply:
Fixed for the whole file.